includes/abstracts/abstract-wc-order.php

```diff
@@ -1599,9 +1599,9 @@ public function get_formatted_order_total() {
 	public function get_subtotal_to_display( $compound = false, $tax_display = '' ) {
 		$tax_display = $tax_display ? $tax_display : get_option( 'woocommerce_tax_display_cart' );
 		$subtotal    = 0;
-
+		$items = apply_filters( 'woocommerce_subtotal_order_item_types', 'line_item' );
```

Comment:
Bug here. Items will be a string, not an array of items.

Reply:
@mikejolley If $types parameter is added as parameter for filter inside get_items() function than this one can be excluded.